examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -9089,6 +9155,117 @@
             }
           ]
         },
+        {
+          "name": "Localization Time Format",
+          "code": 44,
+          "mfgCode": null,
+          "define": "LOCALIZATION_TIME_FORMAT_CLUSTER",
+          "side": "client",
+          "enabled": 0,
+          "commands": [],
+          "attributes": []
+        },
+        {
+          "name": "Localization Time Format",
```

Comment:
Why is this enabled on multiple endpoints?  Is that expected here?

Reply:
Just to pass the Darwin auto-generated test, which use endpoint 1 by default, I can only enable this on endpoint 1

Reply:
See the `asExpectedEndpointForCluster` function in`src/darwin/Framework/CHIP/templates/helper.js` where you can specify which endpoint the test should use.  1 is the default, but we use 0 for a bunch of clusters.